src/chrome/komodo/content/hyperlinks/csscolorpicker.js

```diff
@@ -387,7 +388,13 @@ ko.hyperlinks.ColorPickerHandler.prototype.colorToHex = function(color) {
 ko.hyperlinks.ColorPickerHandler.prototype.colorToRGB = function(color) {
     var span = document.createElement('span');
     span.style.color = color;
-    return window.getComputedStyle(span, null).color;
+    var _color = window.getComputedStyle(span, null).color;
```

Comment:
Nitpick time! This need to be named better, the variable name should reflect what it's holding.